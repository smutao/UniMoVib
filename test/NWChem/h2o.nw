START dft

geometry
 zmatrix
 O
 H1 O OH
 H2 O OH H1 HOH
 variables
  OH    0.99104763
  HOH 104.51804533
 end
end

BASIS SPHERICAL
 H library 3-21g
 O library 3-21g
END

dft
 xc  pbe0
end

TASK dft freq


