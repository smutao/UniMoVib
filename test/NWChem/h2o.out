 argument  1 = 01.nw
                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = schem1a1.systems.smu.edu
    program       = nwchem
    date          = Wed Jan  8 14:01:14 2014

    compiled      = 日_ 4月_10_17:38:54_2011
    source        = /home/chem/source/nwchem-6.0/
    nwchem branch = 6.0
    input         = 01.nw
    prefix        = dft.
    data base     = ./dft.db
    status        = startup
    nproc         =        4
    time left     =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 CS  symmetry detected
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.12132262     0.00000000     0.00000000
    2 H1                   1.0000    -0.48529046     0.78370656     0.00000000
    3 H2                   1.0000    -0.48529046    -0.78370656     0.00000000
 
      Atomic Mass 
      ----------- 
 
      O                 15.994910
      H1                 1.007825
      H2                 1.007825
 

 Effective nuclear repulsion energy (a.u.)       8.8809307880

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
      Symmetry information
      --------------------
 
 Group name             Cs        
 Group number              2
 Group order               2
 No. of unique centers     3
 
      Symmetry unique atoms
 
     1    2    3
 


                                Z-matrix (user)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

  Constrained variables are marked with a plus sign (+).

 
      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1+Stretch      OH          2     1                       0.99105
    2+Stretch      OH          3     1                       0.99105
    3 Bend         HOH         3     1     2               104.51805
 
 
            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.12132262     0.00000000     0.00000000
 H1                   -0.48529046     0.78370656     0.00000000
 H2                   -0.48529046    -0.78370656     0.00000000
 
 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 H1               |   1 O                |     1.87281  |     0.99105
    3 H2               |   1 O                |     1.87281  |     0.99105
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H1               |   1 O                |   3 H2               |   104.52
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/wzou/chemsoft/nwchem/data/libraries/>
  
                      Basis "ao basis" -> "" (spherical)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.44717800E+00  0.156285
  1 S  8.24547000E-01  0.904691
 
  2 S  1.83192000E-01  1.000000
 
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.22037000E+02  0.059239
  1 S  4.84308000E+01  0.351500
  1 S  1.04206000E+01  0.707658
 
  2 S  7.40294000E+00 -0.404453
  2 S  1.57620000E+00  1.221560
 
  3 P  7.40294000E+00  0.244586
  3 P  1.57620000E+00  0.853955
 
  4 S  3.73684000E-01  1.000000
 
  5 P  3.73684000E-01  1.000000
 


 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           3-21g                   2        2   2s
 O                           3-21g                   5        9   3s2p


 
 
                   NWChem Nuclear Hessian and Frequency Analysis
                   ---------------------------------------------
 
 
 
                              NWChem Analytic Hessian
                              -----------------------
 
  itol2e modified to match energy
  convergence criterion.
 
                                 NWChem DFT Module
                                 -----------------
 
 
  Caching 1-el integrals 
 
            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:    13
                     number of shells:     9
          Convergence on energy requested: 1.00D-07
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 1.00D-06
 
              XC Information
              --------------
                         PBE0 Method XC Functional
                     Hartree-Fock (Exact) Exchange  0.250          
          PerdewBurkeErnzerhof Exchange Functional  0.750          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local
 
             Grid Information
             ----------------
          Grid used for XC integration:  fine      
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       70           5.0       590
          H1                  0.35       60           6.0       590
          H2                  0.35       60           6.0       590
          Grid pruning is: on 
          Number of quadrature shells:   190
          Spatial weights used:  Erf1
 
          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

 
      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-11
          AO Gaussian exp screening on grid/accAOfunc:  16
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.36108718
 
      Non-variational initial energy
      ------------------------------

 Total energy =     -75.468308
 1-e energy   =    -120.951898
 2-e energy   =      36.602659
 HOMO         =      -0.427716
 LUMO         =       0.150997
 
   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 Grid_pts file          = ./dft.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     12        Max. recs in file   =    690125


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.96            12958580
          Stack Space remaining (MW):       13.11            13107011

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -75.8518594250 -8.47D+01  5.07D-02  3.70D-01     0.2
 d= 0,ls=0.0,diis     2    -75.8605275371 -8.67D-03  2.25D-02  3.77D-01     0.3
 d= 0,ls=0.0,diis     3    -75.8922833877 -3.18D-02  1.77D-03  3.86D-03     0.3
 d= 0,ls=0.0,diis     4    -75.8925811767 -2.98D-04  1.12D-04  7.18D-06     0.4
 d= 0,ls=0.0,diis     5    -75.8925819683 -7.92D-07  1.45D-05  2.40D-08     0.4
 d= 0,ls=0.0,diis     6    -75.8925819722 -3.99D-09  1.18D-06  1.23D-10     0.5


         Total DFT energy =      -75.892581972242
      One electron energy =     -122.139142422005
           Coulomb energy =       46.610686353237
    Exchange-Corr. energy =       -9.245056691460
 Nuclear repulsion energy =        8.880930787986

 Numeric. integr. density =       10.000000207501

     Total iterative time =      0.3s


 
                       DFT Final Molecular Orbital Analysis
                       ------------------------------------
 
 Vector    1  Occ=2.000000D+00  E=-1.908003D+01
              MO Center=  1.2D-01, -3.0D-15, -1.6D-15, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.982322  1 O  s          
 
 Vector    2  Occ=2.000000D+00  E=-1.004093D+00
              MO Center= -1.0D-01, -5.6D-12, -2.4D-13, r^2= 5.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.691476  1 O  s                  1     -0.226549  1 O  s          
     2      0.214389  1 O  s          
 
 Vector    3  Occ=2.000000D+00  E=-5.088224D-01
              MO Center= -1.2D-01,  3.6D-12,  5.4D-26, r^2= 8.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4     -0.405083  1 O  py                 8     -0.354475  1 O  py         
    12      0.231429  3 H  s                 10     -0.231429  2 H  s          
    13      0.201576  3 H  s                 11     -0.201576  2 H  s          
 
 Vector    4  Occ=2.000000D+00  E=-3.410157D-01
              MO Center=  2.2D-01, -5.2D-13, -5.3D-13, r^2= 6.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7     -0.481829  1 O  px                 6     -0.465153  1 O  s          
     3     -0.435698  1 O  px         
 
 Vector    5  Occ=2.000000D+00  E=-2.717637D-01
              MO Center=  1.2D-01,  7.2D-17,  6.4D-13, r^2= 5.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.628334  1 O  pz                 5      0.525304  1 O  pz         
 
 Vector    6  Occ=0.000000D+00  E= 1.068707D-01
              MO Center= -5.4D-01, -4.2D-11,  1.2D-13, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6     -0.960141  1 O  s                 13      0.804781  3 H  s          
    11      0.804781  2 H  s                  7      0.471474  1 O  px         
     3      0.261617  1 O  px         
 
 Vector    7  Occ=0.000000D+00  E= 1.964594D-01
              MO Center= -4.7D-01,  4.4D-11,  5.6D-25, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11     -1.050583  2 H  s                 13      1.050583  3 H  s          
     8      0.737035  1 O  py                 4      0.365574  1 O  py         
 
 Vector    8  Occ=0.000000D+00  E= 9.423730D-01
              MO Center= -3.5D-01, -9.2D-11,  6.2D-25, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      0.941455  3 H  s                 10     -0.941455  2 H  s          
    13     -0.821245  3 H  s                 11      0.821245  2 H  s          
     8      0.261653  1 O  py                 4      0.207230  1 O  py         
 
 Vector    9  Occ=0.000000D+00  E= 1.004487D+00
              MO Center= -5.0D-01,  9.3D-11, -7.7D-15, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.942503  2 H  s                 12      0.942503  3 H  s          
    11     -0.571594  2 H  s                 13     -0.571594  3 H  s          
     3      0.309465  1 O  px                 7      0.162632  1 O  px         
 
 Vector   10  Occ=0.000000D+00  E= 1.468662D+00
              MO Center=  1.2D-01,  7.2D-17, -2.0D-13, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      1.027230  1 O  pz                 9     -0.967649  1 O  pz         
 
 Vector   11  Occ=0.000000D+00  E= 1.553840D+00
              MO Center=  2.8D-01, -1.2D-12,  4.5D-14, r^2= 7.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7     -1.164762  1 O  px                 3      0.990170  1 O  px         
    10     -0.309666  2 H  s                 12     -0.309666  3 H  s          
     6      0.162388  1 O  s          
 
 Vector   12  Occ=0.000000D+00  E= 1.675033D+00
              MO Center=  1.8D-01,  2.8D-13, -5.8D-26, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8     -1.425069  1 O  py                 4      1.039240  1 O  py         
    13     -0.463770  3 H  s                 11      0.463770  2 H  s          
    12     -0.197837  3 H  s                 10      0.197837  2 H  s          
 
 Vector   13  Occ=0.000000D+00  E= 2.755024D+00
              MO Center= -6.3D-02,  1.2D-12,  1.8D-13, r^2= 7.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.936396  1 O  s                  2     -1.629305  1 O  s          
     7     -0.479253  1 O  px                11     -0.344587  2 H  s          
    13     -0.344587  3 H  s                 10     -0.277223  2 H  s          
    12     -0.277223  3 H  s                  3      0.154100  1 O  px         
 

 center of mass
 --------------
 x =   0.10097481 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           4.420992349264           0.000000000000           0.000000000000
           0.000000000000           2.352290291784           0.000000000000
           0.000000000000           0.000000000000           6.773282641048
 
     Multipole analysis of the density
     ---------------------------------
 
     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -5.000000     -5.000000     10.000000
 
     1   1 0 0     -0.904759     -0.452379     -0.452379      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000      0.000000
 
     2   2 0 0     -4.332178     -3.217352     -3.217352      2.102525
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -3.096787     -3.741727     -3.741727      4.386667
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -5.104310     -2.552155     -2.552155      0.000000
 
  stpr_wrt_fd_from_sq: overwrite of existing file:./dft.hess
 stpr_wrt_fd_dipole: overwrite of existing file./dft.fd_ddipole

 HESSIAN: the one electron contributions are done in       0.0s


 HESSIAN: 2-el 1st deriv. term done in                     0.0s


 HESSIAN: 2-el 2nd deriv. term done in                     0.0s

  stpr_wrt_fd_from_sq: overwrite of existing file:./dft.hess
 stpr_wrt_fd_dipole: overwrite of existing file./dft.fd_ddipole

 HESSIAN: the two electron contributions are done in       0.3s

                                NWChem CPHF Module
                                ------------------
 
 
  scftype          =     RHF 
  nclosed          =        5
  nopen            =        0
  variables        =       40
  # of vectors     =        9
  tolerance        = 0.10D-03
  level shift      = 0.00D+00
  max iterations   =       50
  max subspace     =       90



Iterative solution of linear equations
  No. of variables       40
  No. of equations        9
  Maximum subspace       90
        Iterations       50
       Convergence  1.0D-04
        Start time      1.4


   iter   nsub   residual    time
   ----  ------  --------  ---------
     1      9    4.22D-01       1.6
     2     18    2.13D-01       1.8
     3     27    3.27D-03       2.0
     4     36    1.47D-04       2.2
     5     45    3.19D-06       2.3
 HESSIAN: the CPHF contributions are done
  stpr_wrt_fd_from_sq: overwrite of existing file:./dft.hess
 stpr_wrt_fd_dipole: overwrite of existing file./dft.fd_ddipole
 HESSIAN: the Hessian is done
 
 
  Vibrational analysis via the FX method 
 
  See chapter 2 in "Molecular Vibrations" by Wilson, Decius and Cross
 
  Vib: Default input used 
 
  Nuclear Hessian passed symmetry test 
 


 ---------------------------- Atom information ----------------------------
     atom    #        X              Y              Z            mass
 --------------------------------------------------------------------------
    O        1  2.2926650D-01  1.3600232D-16  0.0000000D+00  1.5994910D+01
    H        2 -9.1706600D-01  1.4809907D+00  0.0000000D+00  1.0078250D+00
    H        3 -9.1706600D-01 -1.4809907D+00  0.0000000D+00  1.0078250D+00
 --------------------------------------------------------------------------




          ----------------------------------------------------
          MASS-WEIGHTED NUCLEAR HESSIAN (Hartree/Bohr/Bohr/Kamu)
          ----------------------------------------------------


              1           2           3           4           5           6           7           8           9
   ----- ----- ----- ----- -----
    1   2.34829D+01
    2   7.33175D-10 3.67037D+01
    3  -1.33974D-11 5.29028D-11 1.09058D-03
    4  -4.67757D+01 5.65874D+01 1.64204D-09 1.80416D+02
    5   3.77818D+01-7.31099D+01-1.40418D-09-1.87974D+02 3.26935D+02
    6  -8.91079D-11-5.73166D-11-2.48220D-03 1.05527D-10 1.40234D-11 2.52795D-03
    7  -4.67757D+01-5.65874D+01 1.60907D-09 5.92887D+00 3.74590D+01-3.20868D-09 1.80416D+02
    8  -3.77818D+01-7.31099D+01 1.35150D-09-3.74590D+01-3.56789D+01-2.69863D-09 1.87974D+02 3.26935D+02
    9  -9.63101D-11-4.12860D-11-2.48220D-03 1.98298D-10-7.21366D-12 7.39011D-03 1.85346D-10 1.71811D-10 2.52795D-03



          -------------------------------------------------
          NORMAL MODE EIGENVECTORS IN CARTESIAN COORDINATES
          -------------------------------------------------
                 (Frequencies expressed in cm-1)

                    1           2           3           4           5           6
 
 Frequency        -11.33       -1.89        1.01        1.78        4.20       17.16
 
           1     0.00000     0.00000    -0.23563    -0.00017    -0.00003     0.00000
           2     0.00000     0.00000     0.00016    -0.23451     0.05439     0.00000
           3     0.00000     0.23606     0.00000     0.00000     0.00000    -0.08243
           4     0.00000     0.00000    -0.23555    -0.01250    -0.56896     0.00000
           5     0.00000     0.00000     0.00023    -0.24405    -0.38597     0.00000
           6    -0.70436     0.23220     0.00000     0.00000     0.00000     0.66498
           7     0.00000     0.00000    -0.23572     0.01216     0.56889     0.00000
           8     0.00000     0.00000     0.00023    -0.24405    -0.38597     0.00000
           9     0.70436     0.23220     0.00000     0.00000     0.00000     0.66498

                    7           8           9
 
 Frequency       1698.49     3498.81     3643.75
 
           1    -0.07077     0.04460     0.00000
           2     0.00000     0.00000    -0.06758
           3     0.00000     0.00000     0.00000
           4     0.56156    -0.35390    -0.41508
           5     0.37553     0.59590     0.53627
           6     0.00000     0.00000     0.00000
           7     0.56156    -0.35390     0.41508
           8    -0.37553    -0.59590     0.53627
           9     0.00000     0.00000     0.00000



 ----------------------------------------------------------------------------
 Normal Eigenvalue ||         Derivative Dipole Moments (debye/angs)
  Mode   [cm**-1]  ||      [d/dqX]             [d/dqY]           [d/dqZ]
 ------ ---------- || ------------------ ------------------ -----------------
    1      -11.335 ||       0.000               0.000             0.000
    2       -1.890 ||       0.000               0.000            -0.015
    3        1.008 ||       0.000               0.000             0.000
    4        1.781 ||       0.000              -0.036             0.000
    5        4.198 ||       0.000              -1.669             0.000
    6       17.160 ||       0.000               0.000             2.833
    7     1698.493 ||       1.027               0.000             0.000
    8     3498.811 ||      -0.437               0.000             0.000
    9     3643.755 ||       0.000               0.013             0.000
 ----------------------------------------------------------------------------



  
  
 ----------------------------------------------------------------------------
 Normal Eigenvalue ||                 Infra Red Intensities
  Mode   [cm**-1]  || [atomic units] [(debye/angs)**2] [(KM/mol)] [arbitrary]
 ------ ---------- || -------------- ----------------- ---------- -----------
    1      -11.335 ||    0.000000           0.000         0.000       0.000
    2       -1.890 ||    0.000009           0.000         0.009       0.002
    3        1.008 ||    0.000000           0.000         0.000       0.000
    4        1.781 ||    0.000057           0.001         0.055       0.010
    5        4.198 ||    0.120799           2.787       117.761      20.794
    6       17.160 ||    0.347987           8.028       339.235      59.901
    7     1698.493 ||    0.045704           1.054        44.554       7.867
    8     3498.811 ||    0.008277           0.191         8.069       1.425
    9     3643.755 ||    0.000008           0.000         0.008       0.001
 ----------------------------------------------------------------------------



 
 
        Vibrational analysis via the FX method 
  --- with translations and rotations projected out ---
  --- via the Eckart algorithm                      ---
 Projected Nuclear Hessian trans-rot subspace norm:7.6106D-33
                         (should be close to zero!) 


From the projected analysis 
The Zero-Point Energy (Kcal/mol) =          12.64274793

 center of mass
 --------------
 x =   0.10097481 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           4.420992349264           0.000000000000           0.000000000000
           0.000000000000           2.352290291784           0.000000000000
           0.000000000000           0.000000000000           6.773282641048

 Rotational Constants
 --------------------
 A=  25.592154 cm-1  ( 36.820578 K)
 B=  13.616892 cm-1  ( 19.591232 K)
 C=   8.887888 cm-1  ( 12.787402 K)


 Temperature                      =   298.15K
 frequency scaling parameter      =   1.0000

 Zero-Point correction to Energy  =   12.633 kcal/mol  (  0.020132 au)
 Thermal correction to Energy     =   14.411 kcal/mol  (  0.022965 au)
 Thermal correction to Enthalpy   =   15.003 kcal/mol  (  0.023909 au)

 Total Entropy                    =   46.623 cal/mol-K
   - Translational                =   34.593 cal/mol-K (mol. weight =  18.0106)
   - Rotational                   =   12.025 cal/mol-K (symmetry #  =        1)
   - Vibrational                  =    0.005 cal/mol-K

 Cv (constant volume heat capacity) =    5.996 cal/mol-K
   - Translational                  =    2.979 cal/mol-K
   - Rotational                     =    2.979 cal/mol-K
   - Vibrational                    =    0.037 cal/mol-K



          -------------------------------------------------
          NORMAL MODE EIGENVECTORS IN CARTESIAN COORDINATES
          -------------------------------------------------
             (Projected Frequencies expressed in cm-1)

                    1           2           3           4           5           6
 
 P.Frequency        0.00        0.00        0.00        0.00        0.00        0.00
 
           1    -0.04961    -0.01898     0.00000     0.00000     0.00000    -0.22957
           2    -0.08157     0.22649     0.00000     0.00000     0.00000    -0.00110
           3     0.00000     0.00000     0.25004     0.00000     0.00000     0.00000
           4    -0.52089    -0.31194     0.00000     0.00000     0.00000    -0.10350
           5    -0.44635    -0.00027     0.00000     0.00000     0.00000     0.09648
           6     0.00000     0.00000     0.00000     0.99611     0.00000     0.00000
           7     0.42167     0.27398     0.00000     0.00000     0.00000    -0.35563
           8    -0.44635    -0.00027     0.00000     0.00000     0.00000     0.09648
           9     0.00000     0.00000     0.00000     0.00000     0.99611     0.00000

                    7           8           9
 
 P.Frequency     1698.49     3498.81     3643.78
 
           1    -0.07077     0.04460     0.00000
           2     0.00000     0.00000    -0.06758
           3     0.00000     0.00000     0.00000
           4     0.56156    -0.35390    -0.41509
           5     0.37553     0.59590     0.53627
           6     0.00000     0.00000     0.00000
           7     0.56156    -0.35390     0.41509
           8    -0.37553    -0.59590     0.53627
           9     0.00000     0.00000     0.00000



 ----------------------------------------------------------------------------
 Normal Eigenvalue ||    Projected Derivative Dipole Moments (debye/angs)
  Mode   [cm**-1]  ||      [d/dqX]             [d/dqY]           [d/dqZ]
 ------ ---------- || ------------------ ------------------ -----------------
    1        0.000 ||       0.000              -1.383             0.000
    2        0.000 ||       0.000              -0.860             0.000
    3        0.000 ||       0.000               0.000            -0.948
    4        0.000 ||       0.000               0.000             1.888
    5        0.000 ||       0.000               0.000             1.888
    6        0.000 ||       0.000               0.370             0.000
    7     1698.493 ||       1.027               0.000             0.000
    8     3498.811 ||       0.437               0.000             0.000
    9     3643.778 ||       0.000               0.013             0.000
 ----------------------------------------------------------------------------



  
  
 ----------------------------------------------------------------------------
 Normal Eigenvalue ||           Projected Infra Red Intensities
  Mode   [cm**-1]  || [atomic units] [(debye/angs)**2] [(KM/mol)] [arbitrary]
 ------ ---------- || -------------- ----------------- ---------- -----------
    1        0.000 ||    0.082893           1.912        80.808      14.269
    2        0.000 ||    0.032031           0.739        31.226       5.514
    3        0.000 ||    0.038946           0.899        37.966       6.704
    4        0.000 ||    0.154525           3.565       150.639      26.599
    5        0.000 ||    0.154525           3.565       150.639      26.599
    6        0.000 ||    0.005931           0.137         5.782       1.021
    7     1698.493 ||    0.045704           1.054        44.554       7.867
    8     3498.811 ||    0.008277           0.191         8.069       1.425
    9     3643.778 ||    0.000008           0.000         0.008       0.001
 ----------------------------------------------------------------------------



 vib:animation  F

 Task  times  cpu:        1.7s     wall:        2.3s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  649      649     1.58e+04 2562     1.10e+04  272        0        0     
number of processes/call 1.33e+00 2.42e+00 1.65e+00 2.60e+00 0.00e+00
bytes total:             3.35e+06 5.07e+05 2.27e+06 7.98e+04 0.00e+00 0.00e+00
bytes remote:            1.49e+06 1.58e+05 1.58e+06 -5.10e+04 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 50728 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        25	        53
	current total bytes		         0	         0
	maximum total bytes		   1188968	  63750016
	maximum total K-bytes		      1189	     63751
	maximum total M-bytes		         2	        64
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        1.8s     wall:        2.4s
